src/xrEngine/Text_Console.cpp

```diff
@@ -25,17 +25,17 @@ CTextConsole::CTextConsole()
 
 CTextConsole::~CTextConsole() { m_pMainWnd = NULL; }
 //-------------------------------------------------------------------------------------------
-LRESULT CALLBACK TextConsole_WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
+LRESULT CALLBACK TextConsole_WndProc(HWND hWnd, unsigned int uMsg, WPARAM wParam, LPARAM lParam);
```

Comment:
Please, return `UINT` for this function.
```suggestion
LRESULT CALLBACK TextConsole_WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
```